assets/scss/lib/partials/_guides.scss

```diff
@@ -0,0 +1,40 @@
+// Links used to navigate between guide pages
+
+.localgov_guides--navigation {
+  a {
+    display: inline-block;
+    @include rem(min-width, 175);
+    position: relative;
+    text-decoration: none;
+
+    span {
+      position: absolute;
+      @include rem (top, 15);
+    }
+
+    &.localgov_guides--next {
+      text-align: right;
+      @include rem(padding-right, 34);
+
+      span {
+        @include rem(right, 7);
+      }
+    }
+
+    &.localgov_guides--previous {
+      text-align: left;
+      @include rem(padding-left, 34);
+
+      span {
+        @include rem(left, 7);
+      }
+    }
+
+    &:first-child {
+      margin-right: 1rem;
+    }
+
+    .btn {
```

Comment:
is this meant to be empty?

Reply:
No, it's a mistake, thanks @cjstevens78. Fixed now.